cli_tools_integ_tests/boot_inspect/boot_inspect_test.go

```diff
@@ -0,0 +1,135 @@
+package boot_inspect
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+
+	"github.com/google/uuid"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/common/disk"
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/daisycommon"
+	daisycompute "github.com/GoogleCloudPlatform/compute-image-tools/daisy/compute"
+)
+
+var (
+	project = getConfig("GOOGLE_CLOUD_PROJECT", "project")
+	zone    = getConfig("GOOGLE_CLOUD_ZONE", "compute/zone")
+
+	wfAttrs = daisycommon.WorkflowAttributes{
+		Project:           project,
+		Zone:              zone,
+		WorkflowDirectory: "../../daisy_workflows",
+	}
+)
+
+func TestBootInspect(t *testing.T) {
+
+	client, err := daisycompute.NewClient(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	for _, tt := range []struct {
+		imageURI string
+		expected disk.InspectionResult
+	}{
+		{
+			"projects/opensuse-cloud/global/images/opensuse-leap-15-2-v20200702",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "opensuse",
+				Major:        "15",
+				Minor:        "2",
+			},
+		}, {
+			"projects/suse-sap-cloud/global/images/sles-15-sp1-sap-v20200803",
+			disk.InspectionResult{
+				Architecture: "x64",
+				Distro:       "sles-sap",
+				Major:        "15",
+				Minor:        "1",
+			},
+		}, {
+			"projects/compute-image-tools-test/global/images/windows-7-ent-x86-nodrivers",
+			disk.InspectionResult{
+				Architecture: "x86",
+				Distro:       "windows",
+				Major:        "6",
+				Minor:        "1",
+			},
+		},
+	} {
+		// Without this, each parallel test will reference the same tc instance.
+		tt := tt
```

Comment:
how does this work?

Reply:
Good question .. added a reference to the golang FAQ (https://github.com/golang/go/wiki/CommonMistakes#using-goroutines-on-loop-iterator-variables)